src/analysisd/format/json_extended.c

```diff
@@ -194,6 +194,33 @@ void W_JSON_ParseGroups(cJSON* root, const Eventinfo* lf)
         }
         token = strtok(0, delim);
     }
+
+    //Add SCA compliance groups
+    cJSON *data = cJSON_GetObjectItem(root,"data");
+    if(data){
+        cJSON *sca = cJSON_GetObjectItem(data,"sca");
+        if(sca){
+            cJSON *check = cJSON_GetObjectItem(sca,"check");
+            if(check){
+                cJSON *compliances = cJSON_GetObjectItem(check,"compliance");
+                cJSON *compliance;
+                cJSON_ArrayForEach(compliance,compliances){
+                    add_SCA_groups(rule, compliance->string, compliance->valuestring);
+                }
+            }
+        }
+    }
+}
+
+void add_SCA_groups(cJSON *rule, char* compliance, char* version){
```

Comment:
For better understanding, the variable `version` can be confusing. In fact, it refers to the value of the compliance control so it would be better to call it `value` IMO.